x-pack/plugins/security/server/anonymous_access/anonymous_access_service.ts

```diff
@@ -166,18 +165,14 @@ export class AnonymousAccessService {
    * anonymous service account credentials.
    */
   private createFakeAnonymousRequest({ authenticateRequest }: { authenticateRequest: boolean }) {
-    return CoreKibanaRequest.from({
+    const fakeRawRequest: FakeRawRequest = {
       headers:
         authenticateRequest && this.httpAuthorizationHeader
           ? { authorization: this.httpAuthorizationHeader.toString() }
           : {},
-      // This flag is essential for the security capability switcher that relies on it to decide if
-      // it should perform a privileges check or automatically disable all capabilities.
```

Comment:
__question:__ it feels like this comment would still be useful to future maintainers since security capability switcher still depends on this flag, or what was the motivation to remove this comment?

Reply:
Fair point. added it back